### DISCUSSION TITLE
TV4Play DRM

### DISCUSSION THREAD

**Discussion by martinbenny** [stefansundin/privatkopiera#228] [Q&A]
Laddade ner Grand Designs, men ett avsnitt verkar de ha DRM-krypterat. Har ni varit med om det tidigare? Varför bara 1 av 5 avsnitt?

https://www.tv4play.se/video/2a0f3995aba3a091baad/avsnitt-4-skuru-kontorshuset-i-sommaridyllen

**Comment by Lunarwalker60** [comment]
Wallander S01E06 & 7 är också krypterade. Allt annat i tre säsonger är OK.--Sent with mail.com Mail appOn 2024-09-11, 13:31 martinbenny ***@***.***> wrote:

  
  Laddade ner Grand Designs, men ett avsnitt verkar de ha DRM-krypterat. Har ni varit med om det tidigare? Varför bara 1 av 5 avsnitt?
  https://www.tv4play.se/video/2a0f3995aba3a091baad/avsnitt-4-skuru-kontorshuset-i-sommaridyllen
